### ISSUE TITLE
How would you recommend that I set the colors of the words?

### ISSUE THREAD

**Issue by mikepsinn** [weihanchen/angular-d3-word-cloud#3]
Thanks for your great work here, @weihanchen !

Do you have any suggestions about how I should set the colors of individual words?

Thanks for your help!  :D

**Comment by weihanchen** [comment]
The library currently uses d3.js schemeCategory20, has not yet implemented your suggestion, and is currently designing the interface of paramater, perhaps the next release will be added the custom color Configuration.

Thanks for your suggestion.

**Comment by mikepsinn** [comment]
Thanks, @weihanchen !

**Comment by weihanchen** [comment]
@mikepsinn
I have been released v0.3.1 contains this feature, you can try the newest version for your suggestion.
Usage like below: 
```html
<div id="wordsCloud">
   <word-cloud words="appCtrl.words" width="appCtrl.width" height="appCtrl.height" padding="5" on-click="appCtrl.wordClicked">
   </word-cloud>
</div>
```
```javascript
(function(){
	angular.module('app',['angular-d3-word-cloud'])
		.controller('appController',['$window','$element',appController])
	function appController($window,$element){
		var self = this;
		self.height = $window.innerHeight * 0.5;
		self.width = $element.find('#wordsCloud')[0].offsetWidth;
		self.wordClicked = wordClicked;
		self.words = [
			{text: 'Angular',size: 25, color: '#6d989e'},
			{text: 'Angular2',size: 35, color: '#473fa3'}
		]

		function wordClicked(word){
			alert('text: ' + word.text + ',size: ' + word.size);
		}
	}
})()
```
[Plunker here](https://plnkr.co/edit/djmNEu?p=preview)

**Comment by mikepsinn** [comment]
That was fast!  :D  Thanks a million, @weihanchen !
